db_migrate/versions/1dd7cd2978aa_deduplicate_files_with_same_content.py

```diff
@@ -0,0 +1,46 @@
+"""deduplicate files with same content
+
+Revision ID: 1dd7cd2978aa
+Revises: 21a4ca1179da
+Create Date: 2017-07-26 13:54:21.400054
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '1dd7cd2978aa'
+down_revision = '21a4ca1179da'
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+from hashlib import sha256
+import zlib
+
+
+def upgrade():
+    content = 'dummy content, backward incompatible update'
+    hasher = sha256()
+    hasher.update(content)
+    content_hash = hasher.hexdigest()
+    content = zlib.compress(content,
+                            zlib.Z_BEST_COMPRESSION)
+    op.drop_column('files', 'content')
+    op.drop_column('files', 'inc_count')
+    op.drop_column('files', 'run_id')
+    op.add_column('files',
+                  sa.Column('content_hash', sa.String(),
```

Comment:
The `FOREIGN KEY` relation is missing from this table, and thus the resulting schema doesn't match the ORM.